Libraries/Components/PasswordBoxWindows/PasswordBoxWindows.windows.js

```diff
@@ -1,41 +1,48 @@
 /**
  * Copyright (c) 2015-present, Facebook, Inc.
- * All rights reserved.
- *
- * This source code is licensed under the BSD-style license found in the
- * LICENSE file in the root directory of this source tree. An additional grant
- * of patent rights can be found in the PATENTS file in the same directory.
  *
```

Comment:
Detailed review needed.

Reply:
RNTester working correctly.

---
In reply to: [169645133](https://github.com/Microsoft/react-native-windows/pull/1659#discussion_r169645133) [](ancestors = 169645133)